Clinical-Genomics/scout
# Ensembl Rest API and Biomart handlers

This PR adds classes for retrieval of data from Ensembl via:
1. Ensembl Rest API (for targeted queries for instance genes, regions etc)
2. Downloading to file (tempfile? Depends on the users requirements) of chunks of information from Biomart (exons for instance). 

Both classes work for retrieving info for build 37 and 38

**How to test**:
1. Check unit tests
1. It is meant to be used from the CLI. 
1. It is meant to fix the issue of not being able of downloading data for build 38 in PR #1295

**Expected outcome**:
Use this code to download exons in build 37 and 38 and it should work!

**Review:**
